data/json/items/comestibles/carnivore.json

```diff
@@ -55,6 +55,16 @@
     "description": "The stomach of a large humanoid creature.  It is surprisingly durable.",
     "proportional": { "weight": 2.0, "volume": 2.0, "price": 1.5, "calories": 2.0 }
   },
+  {
+    "id": "human_fat",
+    "copy-from": "fat",
+    "type": "COMESTIBLE",
+    "cooks_like": "human_cooked",
```

Comment:
no, that's not how you use cooks_like. this will make it so whenever it's in a recipe it uses the calories of human flesh.

Reply:
I was wondering about that.  I was trying to copy how "fat" does it, but I couldn't find an entry for cooks_like.  Could you suggest how I should proceed or refer to some examples for me to look at?